docs/00_what_is_cvmfs.md

```diff
@@ -1,13 +1,147 @@
+with a user experience similar to that of an *on-demand streaming service* for music or video,
+but then for software installations (or other types of data).
 
-## Features
+CernVM-FS was developed specifically for [distributing software](#distributing-software-as-main-use-case),
+and provides several interesting features, including
+[de-duplication of files](#de-duplication-of-files),
+[compression of data](#compression-of-data),
+[multi-level caching](#multi-level-caching),
+and [verification of data integrity](#verification-of-data-integrity),
+and has been proven to scale to hundreds of millions of files and tens of thousands of clients.
+
+
+## Use cases and features
+
+### Distributing software as main use case
+
+CernVM-FS was developed at [CERN](https://home.cern/) to let High Energy Physics (HEP) collaborations
+like the [Large Hadron Collider (LHC) project](https://home.cern/science/accelerators/large-hadron-collider)
+deploy software on the worldwide-distributed computing infrastructure used to run data processing applications,
+but can also be used to distribute large *data* repositories.
+
+The software use case is a particular one, since software often comprises many small files that
+are frequently opened and read as a whole, and frequent look-ups for files in multiple directories are
+triggered when search paths are examined.
+
+### De-duplication of files
+
+Due to the content-addressable storage that is used under the hood, CernVM-FS automatically
+effectively stores the contents of a file only once, even when it is included multiple times
+in a particular repository at different paths.
+
+This can result in a significant reduction in storage capacity that is required to host a large stack of
+many software installations, especially when identical files are spread out across the repository,
+as often happens with particular files like example data files across multiple versions of the same software.
+
+### Compression of data
+
+CernVM-FS [stores file content with compression](https://cvmfs.readthedocs.io/en/stable/cpt-repo.html#compression-and-hash-algorithms),
+on the server, which not only further reduces required storage space but also significantly limits the
+network bandwidth that is used to serve that a stored files in a repository across the network.
+On the client side, the data is transparently decompressed as a part of the POSIX-compatible way in which
+the contents of a CernVM-FS repository is presented under `/cvmfs`.
+
+### Automatic updating of repository contents
+
+...
+
+### Multi-level caching
+
+To reduce the latency when accessing files in a CernVM-FS repository, a multi-level caching hierarchy
+is employed.
+
+First of all, CernVM-FS benefits from the standard Linux kernel file system cache on a client system:
+the content of a file that have been read recently will already be cached in memory by the kernel,
+so that file does not need to be read from disk again.
```

Comment:
The contents and the meta-data. I.e. also most lookups are served from the kernel.
```suggestion
the content of a file that have been read recently will already be cached in memory by the kernel,
so that file does not need to be read from disk again.
```